### PR TITLE
[13.0][MIG] base_search_fuzzy: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT19709

### CODE REVIEW

File: base_search_fuzzy/README.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/server-tools <https://github.com/OCA/server-tools/tree/12.0/base_search_fuzzy>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```
Comment (sergio-teruel) [inline base_search_fuzzy/README.rst:151]: Please generate the new readme...

File: base_search_fuzzy/tests/test_query_generation.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.osv import expression
+from odoo.tests.common import TransactionCase, at_install, post_install
```
Comment (sergio-teruel) [inline base_search_fuzzy/tests/test_query_generation.py]: Use tagged

File: base_search_fuzzy/tests/test_query_generation.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+@at_install(False)
+@post_install(True)
```
Comment (sergio-teruel) [inline base_search_fuzzy/tests/test_query_generation.py]:
```suggestion
@tagged('post_install', '-at_install')
```

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
Changes done
Ping @sergio-teruel

File: base_search_fuzzy/i18n/base_search_fuzzy.pot [hunk 4]
```diff
@@ -0,0 +1,116 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* base_search_fuzzy
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (manuelcalerosolis) [inline base_search_fuzzy/i18n/base_search_fuzzy.pot]: Must be 13.0

**Review by manuelcalerosolis — APPROVED** [review]

File: base_search_fuzzy/i18n/base_search_fuzzy.pot (hunk 4 above)
Comment (ernestotejeda) [reply base_search_fuzzy/i18n/base_search_fuzzy.pot]: done

**Comment by ernestotejeda** [comment]
@sergio-teruel @manuelcalerosolis could you review?

**Comment by ernestotejeda** [comment]
@chienandalu  could you review?

File: .isort.cfg [hunk 5]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-default_section=THIRDPARTY
+known_third_party=psycopg2,setuptools
```
Comment (sergio-teruel) [inline .isort.cfg]: Please make a rebase to get the new isort config.

File: base_search_fuzzy/views/trgm_index.xml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                <field name="index_name" />
+                <field name="index_type" />
+            </tree>
+        </field>
+    </record>
+    <record id="trgm_index_action" model="ir.actions.act_window">
+        <field name="name">Trigram Index</field>
+        <field name="res_model">trgm.index</field>
+        <field name="view_mode">tree,form</field>
+        <field name="type">ir.actions.act_window</field>
```
Comment (sergio-teruel) [inline base_search_fuzzy/views/trgm_index.xml]: You can remove this field

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: .isort.cfg (hunk 5 above)
Comment (ernestotejeda) [reply .isort.cfg]: Current branch 13.0-mig-base_search_fuzzy is up to date.

File: base_search_fuzzy/views/trgm_index.xml (hunk 6 above)
Comment (ernestotejeda) [reply base_search_fuzzy/views/trgm_index.xml]: done

**Comment by sergio-teruel** [comment]
Ok.. thanks

**Review by sergio-teruel — APPROVED** [review]

File: .isort.cfg (hunk 5 above)
Comment (pedrobaeza) [reply .isort.cfg]: Please undo this change and let what is in main branch.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
done @pedrobaeza

**Review by pedrobaeza — COMMENTED** [review]
I'm not seeing in the README the way to check the proper usage. Didn't you talk about some demo data or specifying a way in the README to test it?

**Comment by ernestotejeda** [comment]
@pedrobaeza I added demo data for testing functionally. Do you think the usage description is OK?

File: base_search_fuzzy/README.rst [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+`pg_trgm <https://www.postgresql.org/docs/current/static/pgtrgm.html>`_ module.
+
+
+Usage with demo data
+====================
+
+There are some demo data that allow you to test functionally this module
+if you are in a **demo** database. The steps are the following:
+
+#. Then type the text **Jon Smith** or **Smith John** in the search box and
```
Comment (pedrobaeza) [inline base_search_fuzzy/README.rst:91]:
```suggestion
#. Type the text **Jon Smith** or **Smith John** in the search box and
```

File: base_search_fuzzy/README.rst [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+   select **Search Display Name for: ...**
+#. You will see two contacts, and they are the ones with display names
+   **John Smith** and **John Smizz**. (See corresponding `demo code
+   <https://github.com/OCA/server-tools/tree/13.0/base_search_fuzzy/demo/res_partner_demo.xml#L18>`_)
```
Comment (pedrobaeza) [inline base_search_fuzzy/README.rst:95]: Remove this reference for not having to maintain through code/version changes

**Review by pedrobaeza — COMMENTED** [review]
Almost there

**Comment by ernestotejeda** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**